README.md

```diff
@@ -1,70 +1,137 @@
-# Cluster Version Operator (CVO)
-
-## Building and Publishing CVO
-
-```sh
-./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.sh
+# Cluster Version Operator
+
+The Cluster Version Operator (CVO) is one of the [Cluster Operators](https://github.com/openshift/enhancements/blob/master/dev-guide/operators.md#what-is-an-openshift-clusteroperator)
+that run in every OpenShift cluster. CVO consumes an artifact called a _"release
+payload image,"_ which represents a given version of OpenShift and contains the
+manifests needed for all Cluster Operators to function. CVO reconciles
+the resources in the cluster to match the manifests in the release payload
+image. This way, CVO also implements _cluster upgrades_; after being provided
+a release payload image for a newer OpenShift version, CVO reconciles all
+Cluster Operators to their updated versions, and in turn, the Cluster Operators
```

Comment:
nit: "cluster operators" are our main operands, but we can do other things too, and the CVO is not all that opinionated about what we're managing.  We might want to soften this part a bit like:

> ... the CVO reconciles the incoming manifests, including those for Cluster Operators, bringing the associated Kubernetes resources up to the configuration requested in the incoming release payload.

or some such.

Reply:
Cool. I have softened the paragraph in bf6cc77a13f199601f2f472ce21d6aa2af8d9c1c, also taking into account @obrown1205 's comment below. I think it's generally fine to omit nuances and focus on the main purpose in a README, but I agree I sometimes handwave things too much ;)